cmd/installer/pkg/install/install.go

```diff
@@ -109,7 +109,7 @@ func NewInstaller(cmdline *procfs.Cmdline, seq runtime.Sequence, opts *Options)
 		}
 	}
 
-	i.manifest, err = NewManifest(seq, bootLoaderPresent, i.options)
+	i.manifest, err = NewManifest(seq, i.bootloader.UEFIBoot(), bootLoaderPresent, i.options)
```

Comment:
TODO: above, on line 106, needs a way to force a specific bootloader based on `Options` (flags), so that one could create `sd-boot` metal image for example